command/commands/user.py

```diff
@@ -168,6 +173,13 @@ def edit_helper(self, user_id, param_list):
             edited_user.set_major(param_list["major"])
         if param_list["bio"]:
             edited_user.set_biography(param_list["bio"])
+        if param_list["perm"] and is_admin:
+            edited_user.set_permissions_level(param_list["perm"])
+        elif param_list["perm"] and not is_admin:
+            msg += "\nCannot change user permission: user isn't admin;" +\
+                " this incident will be reported."
+            logging.warn("User {} tried to elevate permissions level."
```

Comment:
Might be valuable to also include which user they tried to elevate permissions on.

Reply:
If the user is not an admin, they aren't allowed to use `--member`, so they can only use this command on themselves. All other instances will result in a "You don't have permissions to do so." error.

Reply:
Er, I don't follow. When will this case even happen, then?

Reply:
When the user self-promotes

Reply:
Oh I see. Perhaps something like "user cannot change own permission" or something? I feel like this message doesn't make it obvious that the user is trying to change their own permission level.